pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -514,38 +513,20 @@ func buildGatewayListenerTLSContext(
 	}
 
 	// Set TLS parameters if they are non-default
-	if len(server.Tls.CipherSuites) > 0 ||
-		server.Tls.MinProtocolVersion != networking.ServerTLSSettings_TLS_AUTO ||
-		server.Tls.MaxProtocolVersion != networking.ServerTLSSettings_TLS_AUTO {
-
-		ctx.CommonTlsContext.TlsParams = &tls.TlsParameters{
-			TlsMinimumProtocolVersion: convertTLSProtocol(server.Tls.MinProtocolVersion),
-			TlsMaximumProtocolVersion: convertTLSProtocol(server.Tls.MaxProtocolVersion),
-			CipherSuites:              filteredCipherSuites(server.Tls.CipherSuites),
-		}
+	if len(server.Tls.CipherSuites) > 0 {
```

Comment:
This was added in https://github.com/maistra/istio/commit/443879e3cb83b724b6cbb0ab582b8bda13742e94#diff-7abd8e9bc7226623690e85b5acc0280def2d72df7960b1324349b08dcb186ca1R492